tests/scenarios/bogus/expected.txt

```diff
@@ -35,8 +35,6 @@ setup(
 
 :: check
 WARNING: patch version component should be .0 for versioning strategy...
-warning: CheckCommand: missing required meta-data: url
-warning: CheckCommand: missing meta-data: either (author and author_email) or (maintainer and maintainer_email) should be supplied
```

Comment:
Minor: latest version of setuptools doesn't complain about metadata by default anymore, only when one runs `check --metadata` (not super relevant to the tests here)